hack/build.sh

```diff
@@ -41,6 +41,11 @@ if [[ -z "${VERSION}" ]]; then
     exit 1
 fi
 
+if [[ -z "${REGISTRY}" ]]; then
```

Comment:
So if the `REGISTRY` is a must-have var when triggering make, is there any documentation needs to be updated?
I quickly when through https://velero.io/docs/main/maintainers/#creating-a-release but didn't find any reference.
What should be set for OSS release that images live in dockerhub?

Reply:
The Makefile has a default value for this variable of "velero" which will result in "velero/velero" being used as the image, but I will also add some extra docs to the release process :+1:

Reply:
I opted not to change this in the end as our GitHub action to push images calls a script directly which then invokes Make to perform the push to Dockerhub. Given that this is the default behaviour that we want for our OSS releases for now, there are no changes necessary to the maintainers documentation for performing a release. We had some [existing docs](https://velero.io/docs/v1.6/build-from-source/#publishing-container-images-to-a-registry) related to building and pushing images to your own registry and these still work as intended. I thought about expanding the docs there to include the binary build instructions, however I don't think this is something we want for most developers and we should encourage them to explicitly set the image if doing local builds.

Reply:
sure